0.21/Dockerfile

```diff
@@ -205,7 +205,8 @@ RUN apk add --no-cache \
         libevent \
         libsodium \
         libstdc++ \
-        libzmq
+        libzmq \
+        sqlite-libs
```

Comment:
Is this definitely the correct package name? You're using `sqlite-libs` here but `sqlite-dev` is used in the build image.

Reply:
Yes, sqlite-dev depends on sqlite-libs, but also contains the C header files, which are only needed at compile-time.